src/components/Switch.js

```diff
@@ -19,6 +19,20 @@ const thumbColorEnabled = Platform.select({
   android: colors.gray20,
   ios: colors.lightestText
 });
+const disabledTrackColor = {
+  ...Platform.select({
+    android: { false: colors.shadow, true: colors.shadow },
+    ios: { false: colors.shadow, true: colors.shadow }
```

Comment:
you also changed iOS but are not showing an iOS screenshot. please add a before and after for both platforms.
iOS was quite good before if i remember correctly, why did you also change that?

Reply:
I have updated the screenshots. 👍